frontend/trip-planner/src/Components/Styles/Login.css

```diff
@@ -0,0 +1,96 @@
+.messages {
+    font-size: 1.15rem;
+    color: #590674;
+    font-weight: bold;
+}
+
+.authStatus {
+    font-size: 1.15rem;
+    color: #590674;
+    font-weight: bold;
+}
+
+.loginHeadline {
+    margin-top: 3rem;
+    font-weight: bold;
+    margin-bottom: 1rem;
+    font-size: 1.5rem;
+    color: #060565;
+}
+
+.login_container {
+    text-align: center;
+}
+
+.loginForm {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+}
+
+.loginForm input {
+    width: 10%;
```

Comment:
avoid using percentage when it's out of the whole screen width/height. 
You don't know which screen the user will have. In the cases you want to limit the width- it's better to use `min-width`/`width` in pixels.